app/livechat/server/lib/Livechat.js

```diff
@@ -763,18 +638,24 @@ export const Livechat = {
 		Rooms.removeByVisitorToken(token);
 	},
 
-	saveDepartment(_id, departmentData, departmentAgents) {
+	saveDepartment(_id, departmentData, departmentAgents, customFields = []) {
 		check(_id, Match.Maybe(String));
 
-		check(departmentData, {
+		const validationFields = {
 			enabled: Boolean,
 			name: String,
 			description: Match.Optional(String),
 			showOnRegistration: Boolean,
 			email: String,
 			showOnOfflineForm: Boolean,
+		};
+
+		customFields.forEach((field) => {
```

Comment:
looking at this again, my code suggestion is not very accurate since it will replace fields already in `validationFields` object.. so please, add this check on the final version.